RaccoonPG.py

```
@@ -0,0 +1,389 @@
+        if self.HP <= 0:
+            self.inCombat == False
+
+    def __str__(self):
+        stats = 'tank, {0}, {1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}, {9}, {10}, {11}, {12}, {13}, {14}, {15}, {16}'.format(self.LVL, self.name, self.HP, self.MP, self.ATTK, self.DEF,  self.MATTK, self.MDEF, self.DODGE, self.SPD, float(self.EXP), float(self.EXPcap), self.POWER, self.CRIT, self.turn, self.inCombat, self.money)
+        return stats
+
+    def deal_damage(self, enemy):
+        damage = Animal.damage_formula(self, enemy, self.POWER, self.LVL, self.ATTK, self.CRIT)
+        enemy.HP -= damage
+        if enemy.HP <= 0:
+            enemy.inCombat == False
```

Ruff (0.14.0)

Avoid equality comparisons to `False`; use `not enemy.inCombat:` for false checks

Replace with `not enemy.inCombat`

(E712)